kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/UserCredentialsForm.vue

```diff
@@ -42,6 +42,14 @@
         :isUniqueValidator="!selectedUser ? uniqueUsernameValidator : () => true"
       />
 
+      <KRouterLink
```

Comment:
I was just passing by... :sweat_smile:. Just wanted to flag that I think we should manage this redirection with the wizard state machine, as if we do this way we will miss the state update on the machine, and there can be errors in the following transitions. I think an error could raise probably if we click on "use an admin account" to import users because it wont be in the right state.

Reply:
This is a good point. I haven't encountered any errors when trying it so far, but redirecting through the wizard state machine would definitely be the safer method to make sure we are accurately updating the state.